docs/en/Running-Hive-with-Alluxio.md

```diff
@@ -72,6 +71,134 @@ FIELDS TERMINATED BY '|'
 LOCATION 'alluxio://master_hostname:port/ml-100k';
 ```
 
+## Create Internal Table from Files Located in Alluxio
+
+Hive can create internal tables from files stored on Alluxio and Hive will manage the lifecycle of internal tables.
+When you drop an internal table, Hive drops the table metadata and data both.
+
+### Hive cli examples
+
+Use the data file in `ml-100k.zip` from
+[http://grouplens.org/datasets/movielens/](http://grouplens.org/datasets/movielens/) as an example.
+
+```bash
+$ bin/alluxio fs mkdir /ml-100k
+$ bin/alluxio fs copyFromLocal /path/to/ml-100k/u.user alluxio://master_hostname:port/ml-100k
+```
+
+Then create an internal table:
+
+```
+hive> CREATE TABLE u_user (
+userid INT,
+age INT,
+gender CHAR(1),
+occupation STRING,
+zipcode STRING)
+ROW FORMAT DELIMITED
+FIELDS TERMINATED BY '|'
+LOCATION 'alluxio://master_hostname:port/ml-100k';
+```
+
+## Use Alluxio as the FileSystem for External tables currently in HDFS
+
+First, transfer your table data which originally in HDFS into Alluxio, assuming that you have put
+your table data in this Alluxio path successfully.
+
+```
+alluxio://master_hostname:port/table/data/path/in/alluxio
+```
+Using the following HiveQL to change the table data location：
+
+```
+hive> alter table TABLE_NAME set location "alluxio://master_hostname:port/table/data/path/in/alluxio";
+```
+
+### Hive cli examples
+
+Create an external table in HDFS:
+
+```
+hive> CREATE EXTERNAL TABLE u_user (
+userid INT,
+age INT,
+gender CHAR(1),
+occupation STRING,
+zipcode STRING)
+ROW FORMAT DELIMITED
+FIELDS TERMINATED BY '|'
+LOCATION 'hdfs://namenode_hostname:port/ml-100k';
+```
+Transfer the data from HDFS into Alluxio：
```

Comment:
yes, just set `alluxio.underfs.address=hdfs://namenode:port/` in `alluxio-site.properties` is sufficient